protocol-units/bridge/cli/src/main.rs

```diff
@@ -1,3 +1,42 @@
-fn main() {
-	println!("Hello, world!");
+use anyhow::Result;
+use clap::{Parser, Subcommand};
+
+mod eth;
+mod movement;
+
+#[derive(Parser)]
+#[command(name = "bridge-cli")]
+#[command(about = "A CLI for interacting with various bridge clients", long_about = None)]
+struct Cli {
+	#[command(subcommand)]
+	command: Commands,
+}
+
+#[derive(Subcommand)]
+enum Commands {
+	Eth {
+		#[command(subcommand)]
+		eth_command: eth::EthCommands,
+	},
+
+	Movement {
+		#[command(subcommand)]
+		movement_command: movement::MovementCommands,
+	},
+}
+
+#[tokio::main]
+async fn main() -> Result<()> {
+	let cli = Cli::parse();
+
+	match &cli.command {
+		Commands::Eth { eth_command } => {
+			eth::run(eth_command).await?;
+		}
+		Commands::Movement { movement_command } => {
+			movement::run(movement_command).await?;
+		}
+	}
+
+	Ok(())
 }
```

Comment:
It appears that the command line interface developed is primarily suited for testing purposes rather than for client use in asset exchange. We will need to change this component accordingly. This might also influence the e2e test. Maybe rename this to a testing cli or something.